### PR TITLE
Added tests for Python 3.13.0-rc.1

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

Note, this PR uses RC1 of Python 3.13.0. Issue #1430 will upgrade this to the GA version of 3.13.0.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -288,23 +296,27 @@ dist_manifest_in_files := \
...
 # Keep in sync with dist_manifest_in_files.
-dist_dependent_files := \
+dist_dependent_files_all := \
     LICENSE.txt \
     README.md \
     README_PYPI.md \
     requirements.txt \
-    $(wildcard *.py) \
+		pyproject.toml \
     $(wildcard $( package_name)/*.py) \
```
Comment (KSchopmeyer) [inline Makefile:312]: Why is pyproject.toml line indented?

File: Makefile [hunk 2]
```diff
@@ -288,23 +296,27 @@ dist_manifest_in_files := \
...
     $(wildcard $(package_name)/*/*.py) \
     $(wildcard $(package_name)/*/*/*.py) \
 
+# The actually used dependency list, which removes the version file. Reason is that the
```
Comment (KSchopmeyer) [inline Makefile:316]: slight english grammar thing. Change to "The dependency list actually used."

File: docs/changes.rst [hunk 3]
```diff
@@ -16,6 +16,10 @@ Released: not yet
 
 * Removed support for Python 2.7, 3.6, and 3.7. (issue #1390)
 
+* Removed setup.py and therefore the possibility to run setup.py as a command,
+  for example to install or test pywbemtools. Running setup.py as a command has
+  been deprecated by Python.
+
```
Comment (KSchopmeyer) [inline docs/changes.rst:22]: Note also that this change added the toml file. i.e. removed setup.py in favor of project.toml

File: dev-requirements.txt [hunk 4]
```diff
@@ -1,12 +1,11 @@
 # Pip requirements file for development dependencies.
 
 
-# Include install dependencies
--r requirements.txt
-
-
 # Direct dependencies for development and indirect dependencies for development
-# that are needed for some reason (must be consistent with minimum-constraints.txt)
+# that are needed for some reason (must be consistent with minimum-constraints-develop.txt)
```
Comment (KSchopmeyer) [inline dev-requirements.txt:5]: above line can say Direct and indirect dependencies for development . . .

File: docs/changes.rst [hunk 5]
```diff
@@ -28,11 +32,34 @@ Released: not yet
...
+  Some files have been renamed:
+  - minimum-constraints.txt to minimum-constraints-develop.txt
+  - .safety-policy-all.yml to .safety-policy-develop.yml
+
+  Removed pywbem/_version_scm.py from git tracking, because it is now
+  dynamically created when building the distribution.
+
+* Development: The pywbem version during development now uses an automatically
+  calculated dev number and the git commit hash, e.g. ``1.4.0a1.dev9+gad875911``.
+  Note that the pywbem version numbers for packages released to Pypi is
```
Comment (KSchopmeyer) [inline docs/changes.rst:58]: Remove word "Note that"

**Review by KSchopmeyer — COMMENTED** [review]
I went through the changes but ran out of time to test it myself since I have to pack for trip now.  I suggest approve with changes.

File: Makefile (hunk 1 above)
Comment (andy-maier) [reply Makefile:312]: Because my editor sometimes uses tabs in a Makefile :-)
Fixed it.

File: Makefile (hunk 2 above)
Comment (andy-maier) [reply Makefile:316]: Changed as proposed.

File: docs/changes.rst (hunk 3 above)
Comment (andy-maier) [reply docs/changes.rst:22]: The addition of pyproject.toml is stated in a separate change log entry in section "Enhancements". This entry here is about the incompatibility caused by removing setup.py. Nevertheless, changed the entry to:
```
* The migration from setup.py to pyproject.toml removed therefore the
  possibility to run setup.py as a command, for example to install or test
  pywbemtools. Note that running setup.py as a command has been deprecated by
  the Python setuptools team.
```

File: dev-requirements.txt (hunk 4 above)
Comment (andy-maier) [reply dev-requirements.txt:5]: These are all direct dependencies but only some indirect dependencies (those that are needed for some reason). So just shortening the sentence as proposed does not make the distinction between all and some. But maybe the original sentence also did not do a good job in distinguishing that.

Changed it to:
```
Direct and necessary indirect dependencies for development (...)
```

File: docs/changes.rst (hunk 5 above)
Comment (andy-maier) [reply docs/changes.rst:58]: Done.

**Pull request merged**